keyboards/planck/keymaps/callum/keymap.c

```diff
@@ -1,331 +1,259 @@
+        SEND_STRING(s);
+    }
+    return false;
 }
 
-void set_qwerty(void) {
-#ifdef AUDIO_ENABLE
-    stop_all_notes();
-    PLAY_SONG(qwerty_song);
-#endif
-    set_single_persistent_default_layer(_QWERTY);
-}
+int cmd_keys_down = 0;
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
     switch (keycode) {
-        case COLEMAK:
-            if (record->event.pressed) {
-                set_colemak();
-            }
-            return false;
-        case QWERTY:
+        // Override the defualt auto shifted symbols to use SEND_STRING See
+        // https://github.com/qmk/qmk_firmware/issues/4072
+        case ampr:
+            return send_string_if_keydown(record, "&");
+        case astr:
+            return send_string_if_keydown(record, "*");
+        case at:
+            return send_string_if_keydown(record, "@");
+        case bsls:
+            return send_string_if_keydown(record, "\\");
+        case circ:
+            return send_string_if_keydown(record, "^");
+        case dlr:
+            return send_string_if_keydown(record, "$");
+        case eql:
+            return send_string_if_keydown(record, "=");
+        case exlm:
+            return send_string_if_keydown(record, "!");
+        case grv:
+            return send_string_if_keydown(record, "`");
+        case hash:
+            return send_string_if_keydown(record, "#");
+        case lbrc:
+            return send_string_if_keydown(record, "[");
+        case lcbr:
+            return send_string_if_keydown(record, "{");
+        case lprn:
+            return send_string_if_keydown(record, "(");
+        case perc:
+            return send_string_if_keydown(record, "%");
+        case pipe:
+            return send_string_if_keydown(record, "|");
+        case plus:
+            return send_string_if_keydown(record, "+");
+        case rbrc:
+            return send_string_if_keydown(record, "]");
+        case rcbr:
+            return send_string_if_keydown(record, "}");
+        case rprn:
+            return send_string_if_keydown(record, ")");
+        case tild:
+            return send_string_if_keydown(record, "~");
+
+        // cmd + cmd -> cmd + ctl
+        case cmd:
             if (record->event.pressed) {
-                set_qwerty();
-            }
-            return false;
-        case SYMB:
-            if (record->event.pressed) {
-                layer_off(_MOUSE);
-                layer_on(_SYMB);
-            } else {
-                layer_off(_SYMB);
-            }
-            return false;
-        case MOVE:
-            if (record->event.pressed) {
-                layer_off(_MOUSE);
-                layer_on(_MOVE);
-            } else {
-                layer_off(_MOVE);
-            }
-            return false;
-        case FUNC:
-            if (record->event.pressed) {
-                layer_off(_MOUSE);
-                layer_on(_FUNC);
+                if (cmd_keys_down == 0) {
```

Comment:
Actually, there is a PR in the works that adds CMD/CTRL swapping.  No need to change this, but I figured you'd want the heads up.

Reply:
Cool! I'll keep an eye out for it :)

Reply:
As somebody that just purchased a mac, I definitely have my eye on that PR :)